### PR TITLE
layers: Label new image/buffer copy VUID

### PR DESCRIPTION
**Pull request by spencer-lunarg**
Updates the new common VUID for 1.3.244 headers

This eliminates the duplicate VUID we had

### CODE REVIEW

**Review by ncesario-lunarg — APPROVED** [review]

File: layers/core_checks/copy_blit_resolve_validation.cpp [hunk 1]
```diff
@@ -2060,10 +2060,10 @@ bool CoreChecks::ValidateCmdCopyImageToBuffer(VkCommandBuffer commandBuffer, VkI
                                   VK_IMAGE_LAYOUT_TRANSFER_SRC_OPTIMAL, func_name, src_invalid_layout_vuid, vuid, &hit_error);
         vuid = is_2 ? "VUID-vkCmdCopyImageToBuffer2-imageOffset-07747" : "VUID-vkCmdCopyImageToBuffer-imageOffset-07747";
         skip |= ValidateCopyBufferImageTransferGranularityRequirements(cb_state, *src_image_state, &region, i, func_name, vuid);
-        vuid = is_2 ? "VUID-VkCopyImageToBufferInfo2-pRegions-07924" : "VUID-vkCmdCopyImageToBuffer-pRegions-07924";
+        vuid = is_2 ? "VUID-VkCopyImageToBufferInfo2-srcSubresource-07967" : "VUID-vkCmdCopyImageToBuffer-srcSubresource-07967";
         skip |= ValidateImageMipLevel(cb_state, *src_image_state, region.imageSubresource.mipLevel, i, func_name,
                                       "imageSubresource", vuid);
-        vuid = is_2 ? "VUID-VkCopyImageToBufferInfo2-imageSubresource-07925" : "VUID-vkCmdCopyImageToBuffer-imageSubresource-07925";
+        vuid = is_2 ? "VUID-VkCopyImageToBufferInfo2-srcSubresource-07968" : "VUID-vkCmdCopyImageToBuffer-srcSubresource-07968";
```
Comment (mikes-lunarg) [inline layers/core_checks/copy_blit_resolve_validation.cpp:2066]: Should these be `VUID-VkCopyImageToBufferInfo2-imageSubresource-07967` / `VUID-vkCmdCopyImageToBuffer-imageSubresource-07967`?

File: layers/core_checks/copy_blit_resolve_validation.cpp [hunk 2]
```diff
@@ -2214,10 +2214,10 @@ bool CoreChecks::ValidateCmdCopyBufferToImage(VkCommandBuffer commandBuffer, VkB
                                   VK_IMAGE_LAYOUT_TRANSFER_DST_OPTIMAL, func_name, dst_invalid_layout_vuid, vuid, &hit_error);
         vuid = is_2 ? "VUID-vkCmdCopyBufferToImage2-imageOffset-07738" : "VUID-vkCmdCopyBufferToImage-imageOffset-07738";
         skip |= ValidateCopyBufferImageTransferGranularityRequirements(cb_state, *dst_image_state, &region, i, func_name, vuid);
-        vuid = is_2 ? "VUID-VkCopyBufferToImageInfo2-pRegions-07924" : "VUID-vkCmdCopyBufferToImage-pRegions-07924";
+        vuid = is_2 ? "VUID-VkCopyBufferToImageInfo2-dstSubresource-07967" : "VUID-vkCmdCopyBufferToImage-dstSubresource-07967";
         skip |= ValidateImageMipLevel(cb_state, *dst_image_state, region.imageSubresource.mipLevel, i, func_name,
                                       "imageSubresource", vuid);
-        vuid = is_2 ? "VUID-VkCopyBufferToImageInfo2-imageSubresource-07925" : "VUID-vkCmdCopyBufferToImage-imageSubresource-07925";
+        vuid = is_2 ? "VUID-VkCopyBufferToImageInfo2-dstSubresource-07968" : "VUID-vkCmdCopyBufferToImage-dstSubresource-07968";
```
Comment (mikes-lunarg) [inline layers/core_checks/copy_blit_resolve_validation.cpp:2220]: Should these be `VUID-VkCopyBufferToImageInfo2-imageSubresource-07967` / `VUID-vkCmdCopyBufferToImage-imageSubresource-07967`?

File: layers/core_checks/copy_blit_resolve_validation.cpp (hunk 1 above)
Comment (spencer-lunarg) [reply layers/core_checks/copy_blit_resolve_validation.cpp:2066]: no, the above use `ValidateImageMipLevel` 

this one uses `ValidateImageArrayLayerRange`

07967 - talks about mipLevel
07968 - talks about arrayLayers

Comment (mikes-lunarg) [reply layers/core_checks/copy_blit_resolve_validation.cpp:2066]: So `VUID-VkCopyImageToBufferInfo2-imageSubresource-07967` / `VUID-vkCmdCopyImageToBuffer-imageSubresource-07967` for the mip levels  and `VUID-VkCopyImageToBufferInfo2-imageSubresource-07968` / `VUID-vkCmdCopyImageToBuffer-imageSubresource-07968` for the array layers then? The current VUIDs containing `srcSubresource` don't exist in the spec.

Comment (spencer-lunarg) [reply layers/core_checks/copy_blit_resolve_validation.cpp:2066]: ooo, I see, there is

- VUID-VkCopyImageToBufferInfo2-imageSubresource-07968
- VUID-vkCmdCopyImage-srcSubresource-07968

ok, let me fix these

Comment (spencer-lunarg) [reply layers/core_checks/copy_blit_resolve_validation.cpp:2066]: applied

```patch
diff --git a/layers/core_checks/copy_blit_resolve_validation.cpp b/layers/core_checks/copy_blit_resolve_validation.cpp
index ededb9d46..fbb8d225b 100644
--- a/layers/core_checks/copy_blit_resolve_validation.cpp
+++ b/layers/core_checks/copy_blit_resolve_validation.cpp
@@ -2060,10 +2060,10 @@ bool CoreChecks::ValidateCmdCopyImageToBuffer(VkCommandBuffer commandBuffer, VkI
                                   VK_IMAGE_LAYOUT_TRANSFER_SRC_OPTIMAL, func_name, src_invalid_layout_vuid, vuid, &hit_error);
         vuid = is_2 ? "VUID-vkCmdCopyImageToBuffer2-imageOffset-07747" : "VUID-vkCmdCopyImageToBuffer-imageOffset-07747";
         skip |= ValidateCopyBufferImageTransferGranularityRequirements(cb_state, *src_image_state, &region, i, func_name, vuid);
-        vuid = is_2 ? "VUID-VkCopyImageToBufferInfo2-srcSubresource-07967" : "VUID-vkCmdCopyImageToBuffer-srcSubresource-07967";
+        vuid = is_2 ? "VUID-VkCopyImageToBufferInfo2-imageSubresource-07967" : "VUID-vkCmdCopyImageToBuffer-imageSubresource-07967";
         skip |= ValidateImageMipLevel(cb_state, *src_image_state, region.imageSubresource.mipLevel, i, func_name,
                                       "imageSubresource", vuid);
-        vuid = is_2 ? "VUID-VkCopyImageToBufferInfo2-srcSubresource-07968" : "VUID-vkCmdCopyImageToBuffer-srcSubresource-07968";
+        vuid = is_2 ? "VUID-VkCopyImageToBufferInfo2-imageSubresource-07968" : "VUID-vkCmdCopyImageToBuffer-imageSubresource-07968";
         skip |= ValidateImageArrayLayerRange(cb_state, *src_image_state, region.imageSubresource.baseArrayLayer,
                                              region.imageSubresource.layerCount, i, func_name, "imageSubresource", vuid);
     }
@@ -2214,10 +2214,10 @@ bool CoreChecks::ValidateCmdCopyBufferToImage(VkCommandBuffer commandBuffer, VkB
                                   VK_IMAGE_LAYOUT_TRANSFER_DST_OPTIMAL, func_name, dst_invalid_layout_vuid, vuid, &hit_error);
         vuid = is_2 ? "VUID-vkCmdCopyBufferToImage2-imageOffset-07738" : "VUID-vkCmdCopyBufferToImage-imageOffset-07738";
         skip |= ValidateCopyBufferImageTransferGranularityRequirements(cb_state, *dst_image_state, &region, i, func_name, vuid);
-        vuid = is_2 ? "VUID-VkCopyBufferToImageInfo2-dstSubresource-07967" : "VUID-vkCmdCopyBufferToImage-dstSubresource-07967";
+        vuid = is_2 ? "VUID-VkCopyBufferToImageInfo2-imageSubresource-07967" : "VUID-vkCmdCopyBufferToImage-imageSubresource-07967";
         skip |= ValidateImageMipLevel(cb_state, *dst_image_state, region.imageSubresource.mipLevel, i, func_name,
                                       "imageSubresource", vuid);
-        vuid = is_2 ? "VUID-VkCopyBufferToImageInfo2-dstSubresource-07968" : "VUID-vkCmdCopyBufferToImage-dstSubresource-07968";
+        vuid = is_2 ? "VUID-VkCopyBufferToImageInfo2-imageSubresource-07968" : "VUID-vkCmdCopyBufferToImage-imageSubresource-07968";
         skip |= ValidateImageArrayLayerRange(cb_state, *dst_image_state, region.imageSubresource.baseArrayLayer,
                                              region.imageSubresource.layerCount, i, func_name, "imageSubresource", vuid);
 ```

**Review by mikes-lunarg — APPROVED** [review]

**Pull request merged**